Main.py

```diff
@@ -717,38 +717,38 @@ def on_flash(event):
                         # Start flashing
                         cwd = os.getcwd()
                         if self._config.flash_mode == 'dryRun':
-                            print("Flash Mode: Dry Run")
+                            print("Flash Mode: Dry run")
                             theCmd = os.path.join(cwd, self._config.firmware_id, "flash-dry-run.bat")
                         elif self._config.flash_mode == 'keepData':
-                            print("Flash Mode: Keep Data")
+                            print("Flash mode: Keep data")
                             theCmd = os.path.join(cwd, self._config.firmware_id, "flash-keep-data.bat")
                         elif self._config.flash_mode == 'wipeData':
-                            print("Flash Mode: Wipe Data")
+                            print("Flash mode: Wipe data")
                             dlg = wx.MessageDialog(None, "You have selected to WIPE data\nAre you sure want to continue?",'Wipe Data',wx.YES_NO | wx.ICON_EXCLAMATION)
                             result = dlg.ShowModal()
                             if result == wx.ID_YES:
                                 pass
                             else:
-                                print("User Canceled Flashing.")
+                                print("User canceled flashing.")
                                 return
                             theCmd = os.path.join(cwd, self._config.firmware_id, "flash-wipe-data.bat")
                         else:
-                            print("Flash Mode: UNKNOWN [%s]" % self._config.flash_mode)
+                            print("Flash mode: UNKNOWN [%s]" % self._config.flash_mode)
                             print("Aborting ...")
                             return
                         os.chdir(self._config.firmware_id)
                         runShell2(theCmd)
                         print("Done!")
                         endFlash = time.time()
-                        print("Flashing Elapsed time: %s"%(endFlash - startFlash,))
+                        print("Flashing elapsed time: %s"%(endFlash - startFlash,))
                         os.chdir(cwd)
                     else:
-                        print("ERROR: You must first select a valid adb device.")
+                        print("ERROR: You must first select a valid ADB device.")
                 else:
-                    print("ERROR: Android Platform Tools must be set.")
+                    print("ERROR: Android platform tools must be set.")
             else:
-                print("ERROR: You must first Prepare a patched Firmware package.")
-                print("       Press the `Prepare Package` Button!")
+                print("ERROR: You must first prepare a patched firmware package.")
```

Comment:
I'll stop commenting on every occurrence of widgets, you get the idea

Reply:
Sure, I was just doing that kinda fast and did not think about context.